### PR TITLE
flake8-self recipe

### PR DESCRIPTION
**Pull request by Korijn**

### CODE REVIEW

**Comment by Korijn** [comment]
Added it back.

File: recipes/flake8_self/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
+    - flake8
+
+test:
+  imports:
+    - flake8_self
+
+about:
+  home: http://github.com/Korijn/{{ name }}
```
Comment (CJ-Wright) [inline recipes/flake8_self/meta.yaml]: Would you mind hard coding this?

**Comment by Korijn** [comment]
Done

**Comment by CJ-Wright** [comment]
This seems reasonable to me.

**Pull request merged**